sites/next.skeleton.dev/src/content/docs/get-started/installation/sveltekit.mdx

```diff
@@ -41,8 +41,8 @@ import ProcessStep from '@components/docs/ProcessStep.astro';
         ## Install Tailwind
         Install Tailwind using [Svelte-Add](https://svelte-add.com/) to automate the process.
         ```console
-        npx @svelte-add/tailwindcss@latest --typography false
-        npm install
+        npx svelte-add@latest tailwindcss
+        # choose to not install typography
```

Comment:
Thanks for the submission @iain3000. We need one of two things here:

1. We need the flag to automatically skip this.
2. Otherwise we should skip the comment. This might make it sound like this incompatible, which is not the case.

A lot of folks just have a misconception about what this plugin is and does so we typically try to avoid it.

Reply:
so I think the svelte-add docs are a bit inconsistent on the syntax for passing options to adders; the website still lists `--typography false` while the cli command itself lists `--tailwindcss-typography false`.

The former doesn't currently work, while the latter does so that's what I've amended it to.

Reply:
Perfect, thanks for the quick turn around!